src/dataverk/datapackage.py

```diff
@@ -36,14 +36,14 @@ def _create_datapackage(self, metadata: dict):
         # set defaults for store and repo when not specified
         metadata['store'] = metadata.get('store', StorageType.LOCAL)
         metadata['repo'] = metadata.get('repo', metadata.get('github-repo', ''))
-        metadata['id'] = self._dp_id
+        metadata['id'] = metadata.get('id', self._dp_id)
         metadata['store_path'] = store_path
         metadata['path'] = path
-        metadata['updated'] = today
-        metadata['version'] = "0.0.1"
-        metadata["views"] = []
-        metadata["resources"] = []
-        metadata["datasets"] = {}
+        metadata['updated'] = metadata.get('updated', today)
+        metadata['version'] = metadata.get('version', '0.0.1')
+        metadata["views"] = metadata.get('views', [])
+        metadata["resources"] = metadata.get('resources', [])
+        metadata["datasets"] = metadata.get('datasets', {})
 
         return metadata
 
```

Comment:
Synes det er bra at det blir mulig å overstyre 'id' og 'versjon'!

Lurer på dette for de andre: 
- Hva er det 'updated' egentlig gjør?
- 'views' og 'resources' blir vel oppdatert med 'add_view' og 'add_resource', etter at datapakkeobjektet er laget? Er det hensiktsmessig at ressurser og figurer kan legges til når datapakkeobjektet lages? 
- Og hva er egentlig funksjonen til 'datasets' om ressurser legges i 'resources'? Trengs den lenger?